src/components/Privacy/Privacy.react.js

```diff
@@ -1,344 +1,308 @@
-import '../Terms/Terms.css';
-import $ from 'jquery';
-import Footer from '../Footer/Footer.react';
+import React from 'react';
 import PropTypes from 'prop-types';
+import Footer from '../Footer/Footer.react';
 import StaticAppBar from '../StaticAppBar/StaticAppBar.react';
-import React, { Component } from 'react';
-
-class Privacy extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      open: false,
-      showOptions: false,
-      anchorEl: null,
-      login: false,
-      signup: false,
-      video: false,
-      openDrawer: false,
-    };
-  }
-
-  componentDidMount() {
-    // Adding title tag to page
-    document.title =
-      'Privacy Policy - SUSI.AI, Open Source Artificial Intelligence for Personal Assistants, Robots, Help Desks and Chatbots';
-    //  Scrolling to top of page when component loads
-    $('html, body').animate({ scrollTop: 0 }, 'fast');
-  }
-
-  showOptions = event => {
-    event.preventDefault();
-    this.setState({
-      showOptions: true,
-      anchorEl: event.currentTarget,
-    });
-  };
-
-  _onReady(event) {
-    // access to player in all event handlers via event.target
-    event.target.pauseVideo();
-  }
+import $ from 'jquery';
```

Comment:
We have been grouping node modules together and app components together, so I guess $ should be shifter just after ```import PropTypes```